renderer.php

```diff
@@ -573,6 +574,11 @@ public function discussion_template($d, $forumtype) {
             $revealed = '<span class="label label-danger">'.$nonanonymous.'</span>';
         }
 
+        $threadContent = '';
+        if (!$hideThreadContent) {
+            $threadContent = '<div class="hsuforum-thread-content" tabindex="0">' . $d->message . '</div>';
```

Comment:
Is there any JS etc that could be broken by removing this div, as opposed to making it empty + display none? If not I think its fine as is.